docs/website/versioned_docs/version-3.0.0/overview/cluster-setup/kubernetes.md

```diff
@@ -16,7 +16,13 @@ This guide is helpful in setting up a development environment intended to be use
 * You have a Kubernetes cluster set up (such as [minikube](https://minikube.sigs.k8s.io/docs/start/))
 * You have admin privileges to the cluster
 
-**Important notes:** `odo` will use the __default__  storage provisioning on your cluster. If it have not been set correctly, see our [troubleshooting guide](/docs/getting-started/cluster-setup/kubernetes#troubleshooting) for more details.
+**Important notes:** `odo` will use the __default__  storage provisioning on your cluster. If it has not been set correctly, see our [troubleshooting guide](/docs/getting-started/cluster-setup/kubernetes#troubleshooting) for more details.
+
+## Installing the Service Binding Operator
```

Comment:
We should add in brackets (OPTIONAL) for this section.

Reply:
It's not optional if user want to use binding.